.gitignore

```diff
@@ -1 +1,6 @@
-vendor
+# Ignoring directories.
+/.idea/
+/vendor/
+# Ignoring files.
+/composer.lock
```

Comment:
Don't put the `composer.lock` into .gitignore

https://getcomposer.org/doc/01-basic-usage.md#commit-your-composer-lock-file-to-version-control

Reply:
Yes, you're right. Let me fix this.